src/sc2laddercore/LadderManager.cpp

```diff
@@ -264,26 +264,53 @@ std::string LadderManager::GetBotCommandLine(const BotConfig &AgentConfig, int G
 	{
 	case Python:
 	{
-		OutCmdLine = "python " + AgentConfig.FileName;
+		OutCmdLine = Config->GetValue("PythonBinary") + " " + AgentConfig.FileName;
+		OutCmdLine += " --GamePort " + std::to_string(GamePort) + " --StartPort " + std::to_string(StartPort) + " --LadderServer 127.0.0.1 --OpponentId " + OpponentId;
+		break;
+	}
+	case Wine:
+	{
+		OutCmdLine = "wine " + AgentConfig.FileName;
+		OutCmdLine += " --GamePort " + std::to_string(GamePort) + " --StartPort " + std::to_string(StartPort) + " --LadderServer 127.0.0.1 --OpponentId " + OpponentId;
+		break;
+	}
+	case Mono:
+	{
+		OutCmdLine = "mono " + AgentConfig.FileName;
+		OutCmdLine += " --GamePort " + std::to_string(GamePort) + " --StartPort " + std::to_string(StartPort) + " --LadderServer 127.0.0.1 --OpponentId " + OpponentId;
+		break;
+	}
+	case CommandCenterLinux:
+	{
+		OutCmdLine = "wine " + Config->GetValue("CommandCenterPath") + " --ConfigFile " + AgentConfig.FileName;
```

Comment:
Why do we need this one? CommandCenter can be compiled under linux without wine.

Reply:
I wasnt aware about that. I would say we leave it out at this point and i test with a linux binary. If native binary can be used, a small os-check needs to be implemented to make sure windows launches ".exe" while linux/darwin does not.

Reply:
I personally implemented support of OS X/Linux in vanilla CommandCenter and I can ensure you, that it works. However, if it is not the vanilla version, it should be simply synced with the latest CommandCenter.

Reply:
So I should take this out then?

Reply:
This has been removed now.